src/pages/workspace/accounting/xero/export/XeroPreferredExporterSelectPage.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback, useMemo } from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type { ListItem } from '@components/SelectionList/types';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections';
+import { getAdminEmployees } from '@libs/PolicyUtils';
+import Navigation from '@navigation/Navigation';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import type { WithPolicyConnectionsProps } from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+
+type CardListItem = ListItem & {
+    value: string;
+};
+
+function XeroPreferredExporterSelectPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {export: exportConfiguration} = policy?.connections?.xero?.config ?? {};
+    const exporters = getAdminEmployees(policy);
+
+    const policyID = policy?.id ?? '';
+    const data: CardListItem[] = useMemo(
+        () =>
+            exporters?.reduce<CardListItem[]>((vendors, vendor) => {
+                if (vendor.email) {
+                    vendors.push({
+                        value: vendor.email,
+                        text: vendor.email,
+                        keyForList: vendor.email,
+                        isSelected: exportConfiguration?.exporter === vendor.email,
+                    });
+                }
+                return vendors;
+            }, []),
+        [exportConfiguration, exporters],
+    );
+
+    const selectExporter = useCallback(
+        (row: CardListItem) => {
+            if (row.value !== exportConfiguration?.exporter) {
+                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.XERO, CONST.XERO_CONFIG.EXPORT, {exporter: row.value});
+            }
+            Navigation.goBack(ROUTES.POLICY_ACCOUNTING_XERO_EXPORT.getRoute(policyID));
+        },
+        [policyID, exportConfiguration],
+    );
+
+    return (
+        <AccessOrNotFoundWrapper
```

Comment:
Use `ConnectionLayout`?

Reply:
I can see we also have a sub-note here? We could still use Connection Layout and only use the subnote in the SelectionList.